### PR TITLE
Issue #299: Fix for "New" button redirect on assistant chat

### PR DESCRIPTION
**Pull request by 3thanHead**
https://github.com/davidmyersdev/octo/blob/2fe0412bf5ca2bef415e912f29d094fe22abdebe/composables/useSoftNavigation.ts#L13-L20

Seems like useSoftNavigation is no longer necessary here. Instead of calling pushRoute on a watcher, we can redirect with Vue Router when chat begins. Removing this allows NuxtLink of "New" button to redirect correctly.

### CODE REVIEW

File: components/Assistant.vue [hunk 1]
```diff
@@ -212,6 +213,7 @@ export default defineComponent({
         for await (const _value of chatModel.value.generate(...messages)) {
           // no-op
         }
+        router.push({ path: `/assistant/conversations/${chatId.value}` })
```
Comment (davidmyersdev) [inline components/Assistant.vue:216]: Hey, this looks like a good fix, but can you move the `useRouter` logic into the `AssistantContainer` component?

Comment (3thanHead) [reply components/Assistant.vue:216]: sweet. getting to this now.

Comment (3thanHead) [reply components/Assistant.vue]: done

**Comment by 3thanHead** [comment]
build passes locally for me, but is failing during CI checks. Looking into this.

**Comment by 3thanHead** [comment]
I see issue now after syncing fork. Solving.

**Pull request merged**